src/test/java/Homework17.java

```diff
@@ -0,0 +1,33 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.annotations.Test;
+public class Homework17 extends BaseTest {
+
+@Test
+    public void addSongToPlaylist() throws InterruptedException {
+    navigateToPage();
+    login("demo@class.com", "te$t$tudent");
+    Thread.sleep(2000);
+    grabASong();
+}
+
+public void grabASong() {
+    WebElement song = driver.findElement(By.xpath("//article[@data-test='song-card']"));
+    WebElement playlist = driver.findElement (By.xpath("//section[@id='playlists']//li[3]"));
+
+    Actions acts = new Actions(driver);
+    acts.clickAndHold(song)
+            .release(playlist)
+            .build()
+            .perform();
+
+    playlist.click();
+   // acts.dragAndDrop(song, playlist).build().perform();
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._